_config/site.yml

```diff
@@ -80,11 +80,53 @@ profiles:
 <% end %>
     push_to_searchisko: true
 
-  drupal:
+  drupal_dev:
     <<: *docker
-    base_url: http://<%= ENV['DRUPAL_HOST_IP'] || 'docker' %>:<%= ENV['DRUPAL_HOST_PORT'] %>/
+    base_url: http://<%= ENV['DRUPAL_HOST_IP']%>:<%= ENV['DRUPAL_HOST_PORT'] %>/
```

Comment:
"localhost" won't work for those who have to run Docker through docker-machine i.e anyone with a mac.

control.rb now attempts to take into account whether you're using docker-machine or not. If yes, it expects a host alias for docker to be set, if not then it assumes you're running docker on localhost. In either case, control.rb sets the value of ENV['DRUPAL_HOST_IP'] correctly, so there is no need for us to use the "docker" alias here.